pkg/server/smtp/handler.go

```diff
@@ -76,8 +80,14 @@ var commands = map[string]bool{
 	"QUIT":     true,
 	"TURN":     true,
 	"STARTTLS": true,
+	"AUTH":     true,
 }
 
+const (
+	usernameChallenge = "VXNlciBOYW1lAA=="
```

Comment:
Please move these up into the other const block (line 25ish), and add comments explaining these.

Reply:
Moved. Comment Added.